includes/gateways/paypal/class-wcs-paypal.php

```diff
@@ -125,8 +125,8 @@ public static function init() {
 	 */
 	public static function get_option( $setting_key ) {
 
-		// Post WC 3.3 PayPal's sandbox and live API credentials are stored separately. When requesting the API keys make sure we return the active keys - live or sandbox depending on the mode.
-		if ( ! wcs_is_woocommerce_pre( '3.3' ) && in_array( $setting_key, array( 'api_username', 'api_password', 'api_signature' ) ) && 'yes' === self::get_option( 'testmode' ) ) {
+		// When requesting the API keys make sure we return the active keys - live or sandbox depending on the mode.
+		if ( in_array( $setting_key, array( 'api_username', 'api_password', 'api_signature' ) ) && 'yes' === self::get_option( 'testmode' ) ) {
```

Comment:
This change is throwing an `in_array()` strict check now that it has changed. 
```suggestion
		if ( in_array( $setting_key, array( 'api_username', 'api_password', 'api_signature' ), true ) && 'yes' === self::get_option( 'testmode' ) ) {
```